deploy/ironic-keepalived-config/image_patch.yaml

```diff
@@ -3,9 +3,23 @@ kind: Deployment
 metadata:
   name: metal3-baremetal-operator
 spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: metal3-keepalived
   template:
+    metadata:
+      labels:
+        name: metal3-keepalived
     spec:
+      hostNetwork: true
```

Comment:
you should not be modifying this in this merge patch

Reply:
Fixed